Configurateur/front/lib/l10n/app_en.arb

```diff
@@ -214,8 +215,8 @@
 	"objectPrice": "Object price",
 	"opinion": "Opinion",
 	"opinionConsultCreate": "Consult and give feedbaks on RISU!",
-	"opinionPage": "Opinion page",
-	"opinionPost": "Post an opinion",
+	"opinionPage": "Post a comment",
```

Comment:
why create a duplicate

Reply:
there is no duplicates, just changing the value